helm/defectdojo/values.yaml

```diff
@@ -381,11 +381,11 @@ redis:
   transportEncryption:
     enabled: false
     params: ''
-  existingSecret: defectdojo-redis-specific
-  secretKey: redis-password
-  password: ""
-  cluster:
-    slaveCount: 1
+  auth:
+    existingSecret: defectdojo-redis-specific
+    existingSecretPasswordKey: redis-password
+    password: ""
+  architecture: standalone
```

Comment:
While in past there was cluster way of working by default just add comment with the reference to bitnami manual. e.g. for more advance options check link.

And add it in doc (upgrade notes), because this are breaking changes during the upgrade. (but it looks like no-one is using redis, so this bug wasn't  detected earlier)